### PR TITLE
Frame implementation

### PR DESCRIPTION
**Pull request by Ruli-AlSe**
### Create Frame class 

 
Create Frame class that should contain important information
about hits, spare and strike for each frame. This information will
be important in the future when the game functionality will be implemented.

 
## Description: Why did you change/implement that?

To store and handle important information for each frame during the game.

## Testing: Help me how can I test or look at the changes?

**Fetch gems:** `bundle install`
**Test:** `bundle exec rspec spec/app/`
**Performance:** `bundle exec rubycritic app/` `bundle exec rubycritic spec/app/`

### CODE REVIEW

File: app/frame.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @score = 0
+    @spare = false
+    @strike = false
+    @position = position
+  end
+
+  def hit1=(value)
+    @strike = true if value == 10
+    @hit1 = value
+    @score += value
```
Comment (ese-varo) [inline app/frame.rb:21]: en tus tres metodos `hit` estas accesando directamente la variable `@score`, ya que estas implementando `attr_accessor` para esta lo correcto seria utilizar el metodo `writer` que genera, por lo que solo escribir: `score += value` sin el `@`

**Review by ese-varo — COMMENTED** [review]
Que tal @Ruli-AlSe te deje una observacion, pero en general tu codigo se ve muy bien. :+1:

File: tmp/rubycritic/app/frame.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
```
Comment (EduardoGHdez) [inline tmp/rubycritic/app/frame.html:1]: Not sure if these `tmp` files are needed.
If not, you can ignore them by adding `tmp/` in your `.gitignore`

File: app/frame.rb (hunk 1 above)
Comment (Ruli-AlSe) [reply app/frame.rb:21]: Cierto, mismo comentario que mi hiciste con el juego de la vida :( gracias Álvaro

File: tmp/rubycritic/app/frame.html (hunk 2 above)
Comment (Ruli-AlSe) [reply tmp/rubycritic/app/frame.html:1]: You're right, they aren't! Thanks!

**Pull request merged**